tests/hipscat_import/conftest.py

```diff
@@ -0,0 +1,71 @@
+import os
+import os.path
+
+import pytest
+from dask.distributed import Client
+
+DATA_DIR_NAME = "data"
+SMALL_SKY_DIR_NAME = "small_sky"
+
+
+@pytest.fixture(scope="session", name="dask_client")
+def dask_client():
+    """Create a single client for use by all unit test cases."""
+    client = Client()
+    yield client
+    client.close()
+
+
+def pytest_collection_modifyitems(items):
```

Comment:
I wasn't aware we could leverage pytest's API this much. This dynamic test setup is awesome!